azure-spring-boot/src/main/java/com/microsoft/azure/spring/autoconfigure/b2c/AADB2CAuthorizationRequestResolver.java

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+package com.microsoft.azure.spring.autoconfigure.b2c;
+
+import org.springframework.security.oauth2.client.registration.ClientRegistrationRepository;
+import org.springframework.security.oauth2.client.web.DefaultOAuth2AuthorizationRequestResolver;
+import org.springframework.security.oauth2.client.web.OAuth2AuthorizationRequestResolver;
+import org.springframework.security.oauth2.core.endpoint.OAuth2AuthorizationRequest;
+import org.springframework.util.Assert;
+
+import javax.servlet.http.HttpServletRequest;
+import java.util.HashMap;
+import java.util.Map;
+
+public class AADB2CAuthorizationRequestResolver implements OAuth2AuthorizationRequestResolver {
+
+    private final AADB2CProperties properties;
+
+    private final OAuth2AuthorizationRequestResolver defaultResolver;
+
+    public AADB2CAuthorizationRequestResolver(ClientRegistrationRepository registrationRepository,
+                                              AADB2CProperties properties) {
+        this.defaultResolver = new DefaultOAuth2AuthorizationRequestResolver(registrationRepository,
+                "/oauth2/authorization");
```

Comment:
Do we need constant variable for `/oauth2/authorization` here?

Reply:
I am not sure why this is required, will investigate it and add const then. As you can see, I marked TODO label here.